dev-dotnet/dotnet-reference-packages/dotnet-reference-packages-1.0.0_p20210210.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit check-reqs
+
+# Disclaimer: While a bunch of generated .il (.net assembly) files can be
+#  considered not to be *source code* per se, none of the files here contain
+#  any actual code. Only symbols/references for a variety of libs.
+
+DESCRIPTION=".NET Source-build Reference Packages"
+HOMEPAGE="https://github.com/dotnet/source-build-reference-packages/"
+
+BOOT_PV="5.0.202"
+BOOT_ARTIFACTS_PV="0.1.0-${BOOT_PV}-1088763-20210414.1"  # See eng/Versions.props
+BOOT_SRC_URI="https://dotnetcli.azureedge.net/dotnet/Sdk/${BOOT_PV}/dotnet-sdk-${BOOT_PV}-linux-x64.tar.gz
+	https://dotnetcli.azureedge.net/source-built-artifacts/assets/Private.SourceBuilt.Artifacts.${BOOT_ARTIFACTS_PV}.tar.gz
```

Comment:
Should this be renamed to something a little more related to the package?

Reply:
I don't know. Any suggestions? Dotnet.Private.SourceBuilt.Artifacts?

Reply:
I would just append `dotnet-` so it's a tad easier to know what it's about